nautobot_circuit_maintenance/__init__.py

```diff
@@ -23,6 +24,22 @@ def custom_field_extension(sender, **kwargs):  # pylint: disable=unused-argument
         field.content_types.set([ContentType.objects.get_for_model(Provider)])
 
 
+def import_notification_sources(sender, **kwargs):  # pylint: disable=unused-argument
+    """Import Notification Sources from Nautobot_configuration.py.
+
+    This is a temporary solution until a proper secrets backend is implemented.
+    For now, we create the aliases in the DB but the secrets are fetched via ENV.
+    """
+    # pylint: disable=import-outside-toplevel
+
+    from nautobot_circuit_maintenance.models import NotificationSource
+
+    for notification_source in settings.PLUGINS_CONFIG.get("nautobot_circuit_maintenance", {}).get(
+        "notification_sources", []
+    ):
+        NotificationSource.objects.get_or_create(alias=notification_source["alias"])
```

Comment:
This logic probably should be refined so that when an entry is removed from `notification_sources` it gets deleted from Nautobot as well?